src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -2336,26 +2336,18 @@ private SyntaxTrivia ScanWhitespace()
 
                 if (width < MaxCachedTokenSize)
                 {
-                    return _cache.LookupTrivia(
-                        TextWindow.CharacterWindow,
-                        TextWindow.LexemeRelativeStart,
-                        width,
-                        hashCode,
-                        CreateWhitespaceTrivia,
-                        TextWindow);
+                    return _cache.LookupWhitespaceTrivia(
+                        TextWindow,
+                        this.LexemeStartPosition,
+                        hashCode);
```

Comment:
moved off of a generic method (that was only ever called here) into a completely reified method.